airbyte-integrations/connectors/source-chartmogul/source_chartmogul/spec.json

```diff
@@ -19,6 +19,13 @@
         "description": "UTC date and time in the format 2017-01-25T00:00:00Z. When feasible, any data before this date will not be replicated.",
         "examples": ["2017-01-25T00:00:00Z"],
         "order": 1
+      },
+      "interval": {
+        "type": "string",
+        "description": "Some APIs such as Metrics requires intervals to cluster data.",
```

Comment:
```suggestion
        "description": "Some APIs endpoints such as Metrics requires intervals to cluster data.",
```

Reply:
Actually disagree with this suggestion @marcosmarxm, the original copy looks good to me. 

My only request here is that we hyperlink ideally to Chartmogul docs which further explain what "intervals" are.